cmd/warpforge/watch.go

```diff
@@ -23,20 +32,132 @@ var watchCmdDef = cli.Command{
 	Action: util.ChainCmdMiddleware(cmdWatch,
 		util.CmdMiddlewareLogging,
 		util.CmdMiddlewareTracingConfig,
+		util.CmdMiddlewareCancelOnInterrupt,
 	),
+	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:  "socket",
+			Usage: "Experimental flag for getting execution status of the watched plot externally via unix socket",
+		},
```

Comment:
I'd almost argue we don't need a flag at all, because why would we ever not enable this?

But at least let's have it default to "on".

Reply:
(The only reason I can think of to not enable this if a user gets stuck in a horrible situation where they absolute cannot create socket files, and they want to try to use the rest of the "watch" feature anyway.  Perhaps possible.  But seems rare.)

Reply:
Agree, should probably be on by default.

Reply:
ehhh... fine.

Reply:
My primary reason for this would be that we might create other kinds of sockets instead of unix sockets.

Reply:
I can see other kinds of sockets being in the future, for sure.  But I still think this one also defaults to on, definitely for now, and probably in the future too even if there are also others.